diff --git a/build/index.js b/build/index.js
index 8bfab87c92f569f78454ba385ee26be5b374b6f3..620e542c1e9a32bc93ba3b1b0621374216f0bf86 100644
--- a/build/index.js
+++ b/build/index.js
@@ -21,7 +21,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 const postcss_selector_parser_1 = __importDefault(require("postcss-selector-parser"));
 const minimal_renamer_1 = require("./minimal-renamer");
 // eslint-disable-next-line no-redeclare
-const plugin = ({ strategy = 'none', by = 'whole', prefix = '', except = [], ids = false, outputMapCallback, } = {}) => {
+const plugin = ({ strategy = 'none', by = 'whole', prefix = '', except = [], ids = false, outputMapCallback, rename = string => string } = {}) => {
     const exceptSet = new Set(except);
     return {
         postcssPlugin: 'postcss-rename',
@@ -31,18 +31,14 @@ const plugin = ({ strategy = 'none', by = 'whole', prefix = '', except = [], ids
             const outputMap = outputMapCallback
                 ? {}
                 : null;
-            let rename;
-            if (strategy === 'none') {
-                rename = name => name;
-            }
-            else if (strategy === 'debug') {
+            if (strategy === 'debug') {
                 rename = name => name + '_';
             }
             else if (strategy === 'minimal') {
                 const renamer = new minimal_renamer_1.MinimalRenamer(skip);
                 rename = name => renamer.rename(name);
             }
-            else if (typeof strategy === 'string') {
+            else if (typeof strategy === 'string' && strategy !== 'custom') {
                 throw new Error(`Unknown strategy "${strategy}".`);
             }
             else {
